modules/sim_2d/sim_2d_component.cc

```diff
@@ -100,93 +96,69 @@ void SimControl::InitTimerAndIO() {
       node_->CreateWriter<PredictionObstacles>(FLAGS_prediction_topic);
 
   // Start timer to publish localization and chassis messages.
-  sim_control_timer_.reset(new cyber::Timer(
-      kSimControlIntervalMs, [this]() { this->RunOnce(); }, false));
-  sim_prediction_timer_.reset(
-      new cyber::Timer(kSimPredictionIntervalMs,
-                       [this]() { this->PublishDummyPrediction(); }, false));
+  sim_2d_timer_.reset(new cyber::Timer(
+      kSim2DComponentIntervalMs, [this]() { this->SimStep(); }, false));
+  sim_prediction_timer_.reset(new cyber::Timer(
+      kSimPredictionIntervalMs, [this]() { this->PublishDummyPrediction(); },
+      false));
 }
 
-void SimControl::Init(bool set_start_point, double start_velocity,
-                      double start_acceleration) {
+void Sim2DComponent::InitState(bool set_start_point, double start_velocity,
+                               double start_acceleration) {
   if (set_start_point && !FLAGS_use_navigation_mode) {
     InitStartPoint(start_velocity, start_acceleration);
   }
+  InitTimerAndIO();
```

Comment:
InitTimerAndIO() shall be done once while InitState can be invoked multiple times. I think it is more appropriate to move InitTimerAndIO() to Init().